Dockerfile

```diff
@@ -28,10 +28,10 @@ RUN apt update -q \
        ansible==5.7.1 \
        ansible-core==2.12.5 \
        cryptography==3.4.8 \
-       jinja2==2.11.3 \
+       jinja2==3.1.2 \
        netaddr==0.7.19 \
        jmespath==1.0.1 \
-       MarkupSafe==1.1.1 \
+       MarkupSafe==2.1.2 \
        ruamel.yaml==0.17.21 \
     && KUBE_VERSION=$(sed -n 's/^kube_version: //p' roles/kubespray-defaults/defaults/main.yaml) \
```

Comment:
HI @tu1h 

should we change the `netaddr` and `netaddr` to the version same as requirements :-)